packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -14,6 +14,9 @@ import uuid from 'uuid';
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
 
+const DEFAULT_CLUSTER =
+  process.env.DEFAULT_CLUSTER || 'urn:TEAM:us-east-2_a:identityLookup';
```

Comment:
this is just for convo?

Reply:
If this is just for Convo, and has no impact on any other service clusters, could you add details on what it is within the plugin's `README.md`? I believe the `@webex/test-users` package may have the most up-to-date `README.md` example for environment variables.